network.config.json

```diff
@@ -1,4 +1,6 @@
 {
   "xdcparentnet": "https://devnetstats.apothem.network/devnet",
+  "xdcdevnet": "https://devnetstats.apothem.network/devnet",
```

Comment:
why there have 2 lines

Reply:
xdcdevnet fixed string
xdcparentnet the URL can be replaced during the deploy process for customization. Use devnet URL as default.

Reply:
i think if some parameters no useful , it can be deleted 

can you explain the 4 parameters work?

Reply:
deploy devnet
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdcdevnet
deploy testnet
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdctestnet
deploy custom url
    npx hardhat run scripts/proxy/ProxyGatewayDeploy.js --network xdcparentnet
 
The xdcdevnet and xdctestnet are fixed url not parameter. This just simplifies when you want to deploy different network. Don't need to keep editing config. Also xdcparentnet, xdcsubnet will be used in subnet generator.